### PR TITLE
[NO TICKET] Lint fixes only

### PR DESCRIPTION
**Pull request by sdbars**
This ticket addresses several the lint errors that popped up when running cmd `yarn lint`.

### CODE REVIEW

File: src/styles/components/_Header.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 // Add `a` to give a higher specificity. This fixes the grid utility classes taking precedence
 // over this class, which results in `flex-shrink` not getting applied on small width screen.
-a.hc-c-logo-link {
+.hc-c-logo-link.ds-l-col.ds-l-col--auto {
```
Comment (nichia) [inline src/styles/components/_Header.scss:19]: Can you talk more about this change? The issue that it addresses?

**Review by nichia — COMMENTED** [review]
Noticed a slight color change here, is this an intended change?
Screen Shot 2021-04-08 at 1 39 00 PM

File: src/styles/components/_Header.scss (hunk 1 above)
Comment (sdbars) [reply src/styles/components/_Header.scss:19]: - The lint considers the format a.className a violation.  Refer to this article for more info:  https://nicedoc.io/stylelint/stylelint/blob/master/lib/rules/selector-no-qualifying-type/README.md. 

- I'm going to try this format a > .className and see if it does what's expected. The color for the Log in should be blue.

**Review by pwolfert — COMMENTED** [review]
I didn't realize there were so many open lint violations. Thanks for fixing them! Could you separate the delinting changes into a separate PR? That would make all the changes easier to track and would keep this one focused on the style change. The new PR can use `[NO-TICKET]` in the title instead of being directly linked to this ticket.

File: src/styles/components/_Header.scss (hunk 1 above)
Comment (sdbars) [reply src/styles/components/_Header.scss:19]: the a> .className format did not work and left the css selector as originally submitted

**Comment by nichia** [comment]
Yes, thanks for fixing the lint error, I like Patrick's suggestion to move the lint violations fixes to another PR so that we can track the changes for the Menu style updates ticket easier.

File: src/styles/components/_Header.scss [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 // Add `a` to give a higher specificity. This fixes the grid utility classes taking precedence
 // over this class, which results in `flex-shrink` not getting applied on small width screen.
```
Comment (nichia) [inline src/styles/components/_Header.scss:18]:
```suggestion
// Add `ds-l-col.ds-l-col--auto` to give a higher specificity. This fixes the grid utility classes taking precedence
// over this class, which results in `flex-shrink` not getting applied on small width screen.
```

Comment (sdbars) [reply src/styles/components/_Header.scss:18]: will update the comment as suggested

**Review by nichia — APPROVED** [review]
LGTM

**Pull request merged**